scripts/lib/mkcloud-libvirt.sh

```diff
@@ -112,8 +112,10 @@ done
 iptables_unique_rule PREROUTING -t nat -p tcp --dport 6080 \\
     -j DNAT --to-destination $net_public.2
 
-iptables_unique_rule FORWARD -d $net_admin.0/24 -j ACCEPT
-iptables_unique_rule FORWARD -d $net_public.0/24 -j ACCEPT
+for x in D I ; do
```

Comment:
On boot you do not have working cloud networks anyway, because the libvirt-managed bridge and the public vlan ontop of it are not there, but the restartcloud step takes care of putting all the pieces back together in the right order.
That means, we could even drop the part about adding lines to boot.local. (but that is a different PR to be done on another day)

apart from that, I am rather confident, because it worked in one cloud after using the libvirt_prepare step on my patched version. 
Before that, I could not ssh -p 1122